(venv) root@srv777747:/home/raotsuplementos/htdocs/raotsuplementos.com.mx# mkdir -p templates/checkout/email
(venv) root@srv777747:/home/raotsuplementos/htdocs/raotsuplementos.com.mx# mkdir -p static/images
(venv) root@srv777747:/home/raotsuplementos/htdocs/raotsuplementos.com.mx# python manage.py migrate
Traceback (most recent call last):
  File "/home/raotsuplementos/htdocs/raotsuplementos.com.mx/manage.py", line 23, in <module>
    main()
  File "/home/raotsuplementos/htdocs/raotsuplementos.com.mx/manage.py", line 20, in main
    execute_from_command_line(sys.argv)
  File "/home/raotsuplementos/htdocs/raotsuplementos.com.mx/venv/lib/python3.12/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
    utility.execute()
  File "/home/raotsuplementos/htdocs/raotsuplementos.com.mx/venv/lib/python3.12/site-packages/django/core/management/__init__.py", line 436, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/raotsuplementos/htdocs/raotsuplementos.com.mx/venv/lib/python3.12/site-packages/django/core/management/base.py", line 416, in run_from_argv       
    self.execute(*args, **cmd_options)
  File "/home/raotsuplementos/htdocs/raotsuplementos.com.mx/venv/lib/python3.12/site-packages/django/core/management/base.py", line 460, in execute
    output = self.handle(*args, **options)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/raotsuplementos/htdocs/raotsuplementos.com.mx/venv/lib/python3.12/site-packages/django/core/management/base.py", line 107, in wrapper
    res = handle_func(*args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/raotsuplementos/htdocs/raotsuplementos.com.mx/venv/lib/python3.12/site-packages/django/core/management/commands/migrate.py", line 114, in handle  
    executor = MigrationExecutor(connection, self.migration_progress_callback)  
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^  
  File "/home/raotsuplementos/htdocs/raotsuplementos.com.mx/venv/lib/python3.12/site-packages/django/db/migrations/executor.py", line 18, in __init__
    self.loader = MigrationLoader(self.connection)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/raotsuplementos/htdocs/raotsuplementos.com.mx/venv/lib/python3.12/site-packages/django/db/migrations/loader.py", line 58, in __init__
    self.build_graph()
  File "/home/raotsuplementos/htdocs/raotsuplementos.com.mx/venv/lib/python3.12/site-packages/django/db/migrations/loader.py", line 235, in build_graph
    self.applied_migrations = recorder.applied_migrations()
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/raotsuplementos/htdocs/raotsuplementos.com.mx/venv/lib/python3.12/site-packages/django/db/migrations/recorder.py", line 89, in applied_migrations 
    if self.has_table():
       ^^^^^^^^^^^^^^^^
  File "/home/raotsuplementos/htdocs/raotsuplementos.com.mx/venv/lib/python3.12/site-packages/django/db/migrations/recorder.py", line 63, in has_table
    with self.connection.cursor() as cursor:
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/raotsuplementos/htdocs/raotsuplementos.com.mx/venv/lib/python3.12/site-packages/django/utils/asyncio.py", line 26, in inner
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/raotsuplementos/htdocs/raotsuplementos.com.mx/venv/lib/python3.12/site-packages/django/db/backends/base/base.py", line 320, in cursor
    return self._cursor()
           ^^^^^^^^^^^^^^
  File "/home/raotsuplementos/htdocs/raotsuplementos.com.mx/venv/lib/python3.12/site-packages/django/db/backends/dummy/base.py", line 20, in complain
    raise ImproperlyConfigured(
django.core.exceptions.ImproperlyConfigured: settings.DATABASES is improperly configured. Please supply the ENGINE value. Check settings documentation for more details.